azure-connector/build.gradle.kts

```diff
@@ -0,0 +1,284 @@
+import com.github.spotbugs.snom.SpotBugsTask
+
+/*
+ * Copyright 2024 Aiven Oy
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    id("aiven-apache-kafka-connectors-all.java-conventions")
+}
+group = "io.aiven"
+
+// TODO: document why we stick to these versions
+val kafkaVersion by extra("3.6.0")
```

Comment:
I doubt we should stick to 3.6.0. So far we were trying to keep this version as low as possible.

Reply:
is it okay to use kafka version 3.4, 1 is to old right

Reply:
I would prefer to use the 1.1.0 as other connectors. And migrate to newer in one go.

Reply:
sure will change it to 1.1.0